example/app/example-app/projections.go

```diff
@@ -0,0 +1,327 @@
+	"github.com/crypto-com/chain-indexing/projection/validator"
+	"github.com/crypto-com/chain-indexing/projection/validatorstats"
+)
+
+func initProjections(
+	logger applogger.Logger,
+	rdbConn rdb.Conn,
+	config *bootstrap.Config,
+) []projection_entity.Projection {
+	if config.System.Mode == bootstrap.SYSTEM_MODE_API_ONLY {
+		return []projection_entity.Projection{}
+	}
+
+	var cosmosAppClient cosmosapp.Client
+	if config.CosmosApp.Insecure {
+		cosmosAppClient = cosmosapp_infrastructure.NewInsecureHTTPClient(
+			config.CosmosApp.HTTPRPCUrl, config.Blockchain.BondingDenom,
+		)
+	} else {
+		cosmosAppClient = cosmosapp_infrastructure.NewHTTPClient(
+			config.CosmosApp.HTTPRPCUrl, config.Blockchain.BondingDenom,
+		)
+	}
+
+	projections := make([]projection_entity.Projection, 0, len(config.Projection.Enables))
+	initParams := InitProjectionParams{
+		Logger:  logger,
+		RdbConn: rdbConn,
+
+		CosmosAppClient:       cosmosAppClient,
+		AccountAddressPrefix:  config.Blockchain.AccountAddressPrefix,
+		ConsNodeAddressPrefix: config.Blockchain.ConNodeAddressPrefix,
+
+		GithubAPIUser:    config.GithubAPI.Username,
+		GithubAPIToken:   config.GithubAPI.Token,
+		MigrationRepoRef: config.GithubAPI.MigrationRepoRef,
+	}
+	for _, projectionName := range config.Projection.Enables {
+		projection := InitProjection(
+			projectionName, initParams,
+		)
+		if projection == nil {
+			continue
+		} else {
+			if onInitErr := projection.OnInit(); onInitErr != nil {
+				logger.Errorf(
+					"error initializing projection %s, system will attempt to initialize the projection again on next restart: %v",
+					projection.Id(), onInitErr,
+				)
+				continue
+			}
+		}
+		projections = append(projections, projection)
+	}
+
+	// Append additional projections
+	for _, projectionName := range config.Projection.Enables {
+		projection := InitAdditionalProjection(
+			projectionName, initParams,
+		)
+		if projection == nil {
+			continue
+		} else {
+			if onInitErr := projection.OnInit(); onInitErr != nil {
+				logger.Errorf(
+					"error initializing additional projection %s, system will attempt to initialize the projection again on next restart: %v",
+					projection.Id(), onInitErr,
+				)
+				continue
+			}
+		}
+		projections = append(projections, projection)
+	}
+
+	logger.Infof("Enabled the follow projections: [%s]", strings.Join(config.Projection.Enables, ", "))
+
+	return projections
+}
+
+func InitAdditionalProjection(name string, params InitProjectionParams) projection_entity.Projection {
+	connString := params.RdbConn.(*pg.PgxConn).ConnString()
+
+	githubMigrationHelperConfig := github_migrationhelper.Config{
+		GithubAPIUser:    params.GithubAPIUser,
+		GithubAPIToken:   params.GithubAPIToken,
+		MigrationRepoRef: params.MigrationRepoRef,
+		ConnString:       connString,
+	}
+
+	switch name {
+	case "Example":
+		sourceURL := github_migrationhelper.GenerateDefaultSourceURL("Example", githubMigrationHelperConfig)
```

Comment:
The `sourceURL` is wrong here. Might need to handle custom projection in a different way.

Maybe refer to the code we used in `chain-indexing-server`: https://github.com/crypto-com/chain-indexing-server/blob/ee1e65dc5f0441eba6f07f71bf842e7de05e919f/app/chain-indexing-app/projections.go#L284

Reply:
Thanks and good catch! Updated custom GitHub path